### DISCUSSION TITLE
Why are paths exchanged for numbers?

### DISCUSSION THREAD

**Discussion by BenjaminWaye** [parcel-bundler/parcel#6347] [Q&A] [answered]
Trying to understand more about Parcel and noticed an interesting thing when logging the Less using PostCSS I get random numbers in place of the paths to image assets instead of the actual paths.

Why is this? Is there a way to prevent Parcel from replacing the paths?

See below:

```
url("2ce1c0c3f736f359c8fa887822ab2f74") format("embedded-opentype"), url("de7ac21c892cf0acc7df93e160b2756a") format("woff")
```

Original Less contents:
```
url("../../../dir/dir/dir/dir/font.eot?5b11327f00eef2cc088c20508f1c7541?#iefix") format("embedded-opentype"),
url("../../../dir/dir/dir/dir/font.woff?5b11327f00eef2cc088c20508f1c7541") format("woff");
}
```

**Comment by mischnic** [comment, marked as answer]
Parcel replaces "dependencies", so the argument of `url`, or also `<a href="...">` with a hash. At the very end, that hash is replaced with the actual path to that bundle, which isn't known yet when for example the Less or PostCSS transformers run.

**Reply by BenjaminWaye** [reply]
Interesting! So how does one go about replacing the assets paths in the css files?

**Reply by mischnic** [reply]
What do you mean by "replacing"? Your code (transformer) should probably simply run before these paths are replaced with hashes

**Reply by BenjaminWaye** [reply]
I consider this question answered so I created a new question here https://github.com/parcel-bundler/parcel/discussions/6349
